design/PERSISTENCE.md

```diff
@@ -1,6 +1,27 @@
 # Persistence design
```

Comment:
We should clarify this is about consumer persistence, not producer persistence.

Reply:
It is about the producer, not consumer persistence. I think that is what you meant. But kind of makes the point :D